demos/magic_distillation_demo.ipynb

```diff
@@ -281,7 +281,7 @@
                 "\n",
                 "exact_T0_state = T0_state()\n",
                 "\n",
-                "xs = jnp.linspace(0, 1, 20)\n",
+                "xs = jnp.linspace(0, 1, 5)\n",
```

Comment:
Does the plot still look reasonable with this change?

Reply:
Fixed!